@kiva/kv-components/vue/stories/KvChip.stories.js

```diff
@@ -0,0 +1,22 @@
+import KvChip from '../KvChip.vue';
+
+export default {
+	title: 'KvChip',
+	component: KvChip,
+};
+
+const story = (args) => {
+	const template = (_args, { argTypes }) => ({
+		props: Object.keys(argTypes),
+		components: { KvChip },
+		template: `
+			<kv-chip
+			:title="chip-title"
+			/>
+		`,
+	});
+	template.args = args;
+	return template;
+};
+
+export const Default = story({ title: 'Chip Title' });
```

Comment:
Please add at least 1 more option with a longer title and with an email domain as well as a full email address.